src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -742,8 +743,28 @@ private boolean testAndRunGodMode(String filePath, String content) {
             return false;
         }
     }
-
-    private String processPageSourceForGodMode(String content) {
+    
+    public static String processPageSourceForFailureCase(String content) {
+        Date now = new Date();
+        return processPageSourceForGodMode(content)
+                // jQuery local
+                .replace(Sanitizer.sanitizeForHtml("/js/lib/jquery-ui.min.js"), 
+                         Sanitizer.sanitizeForHtml("{*}/jquery-ui.min.js"))
```

Comment:
oh my brain must have been not working at that time, creating an issue now :sweat: